lib/CL/devices/devices.c

```diff
@@ -514,7 +514,13 @@ pocl_init_devices ()
 #ifdef __linux__
 
 #ifdef ENABLE_HOST_CPU_DEVICES
+/* disabled by default if ENABLE_CONFORMANCE=OFF as it might
```

Comment:
Can you update the documentation of POCL_SIGFPE_HANDLER with this logic?

Reply:
added